webpack.config.js

```diff
@@ -3,6 +3,16 @@ const { resolve } = require('path');
 const webpack = require('webpack');
 
 module.exports = {
+  context: resolve(__dirname, 'src'),
+  devtool: 'cheap-module-eval-source-map',
+  devServer: {
+    hot: true,
+    contentBase: resolve(__dirname, 'dist'),
+    publicPath: '/'
+  },
+  resolve: {
+    extensions: ['.js', '.scss'],
+  },
```

Comment:
I change the order. Now, it doesn't depend on webpack document.

the rule is ...

```

// [a-z] && exclude 4 major key.

entry,
output,
module,
plugins,
```

I think option keys that they're easy to be skipped.